package.json

```diff
@@ -48,6 +48,7 @@
         "js-yaml": "3.13.1",
         "sync-request": "6.1.0",
         "user": "^0.0.0",
+        "utils": "^0.3.1",
```

Comment:
Why is it needed? Are you using it anywhere?

Reply:
Removed